src/Reporter/GitlabReporter.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfTwig\Twigcs\Reporter;
+
+use FriendsOfTwig\Twigcs\Validator\Violation;
+use JsonException;
+use Symfony\Component\Console\Output\OutputInterface;
+use function hash;
+use function implode;
+use function json_encode;
+
+class GitlabReporter implements ReporterInterface
```

Comment:
@frankdekker 

Is there a need to extend from this class or can we mark it as `final`?

Perhaps the class should also be named `GitLabReporter` instead of `GitlabReporter`? After all, the platform is called **GitLab**.